release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -550,6 +550,11 @@ The most common use case for the `networkConfig` configuration setting is to set
 
 For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-host-network-interfaces-in-the-install-config.yaml-file_ipi-install-installation-workflow[Configuring host network interfaces in the install-config.yaml file].
 
+[id="ocp-4-10-networking-k8s-nmstate-operator"]
+==== Kubernetes NMState Operator is GA for bare metal installations
```

Comment:
**thought:** 🤔 Looks like an adjective to me, so maybe use `bare-metal` in both instances. PS: https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal-adj
However, I'll leave it up to you to check existing docs practices and use the correct one.